forgot_password.rb

```diff
@@ -0,0 +1,36 @@
+require 'digest'
+
+def store_new_password(filename, string)
+  File.open(filename, "a+") do |file|
```

Comment:
zamiast robić `truncate`, możesz zrobić `File.open(filename, "w")` - to wtedy wyczyści ten plik